include/matx/generators/random.h

```diff
@@ -546,6 +552,9 @@ template <typename T, int RANK> class randomTensorView_t {
             curandGenerateNormalDouble(gen_, &tmp[1], 1, beta_, alpha_);
           }
         }
+        else {
+          val = 0;
```

Comment:
Do we want to return zero versus throwing an error? I would assume something is broken if I call a random generator and get all zeros.

Reply:
This is device code, so we can't really throw an error here